nle/nethack/actions.py

```diff
@@ -153,7 +153,15 @@ class Command(enum.IntEnum):
     SAVE = ord("S")  # save the game and exit
     SEARCH = ord("s")  # search for traps and secret doors
     SEEALL = ord("*")  # show all equipment in use
+    SEEAMULET = ord('"')  # show the amulet currently worn
+    SEEARMOR = ord("[")  # show the armor currently worn
+    SEEGOLD = ord("$")  # count your gold
+    SEERINGS = ord("=")  # show the ring(s) currently worn
+    SEESPELLS = ord("+")  # list and reorder known spells
+    SEETOOLS = ord("(")  # show the tools currently in use"
     SEETRAP = ord("^")  # show the type of adjacent trap
+    SEEWEAPON = ord(")")  # show the weapon currently wielded
+    SHELL = ord("!")  # do a shell escape (not enabled in NLE build)
```

Comment:
Can we just leave this out 
```suggestion
```

Reply:
If we do that we also need to add `!` to the not-available ASCII characters in the action test this PR adds.

I guess this depends on whether we want to be able to name something with `!`. I guess we don't.